storage/src/test/java/org/apache/kafka/storage/internals/log/TransactionIndexTest.java

```diff
@@ -245,7 +246,7 @@ public void testDoNotCreateFileUntilNeeded() throws IOException {
     }
 
     @Test
-    void testAppendAndCollectAfterClose() throws IOException {
+    public void testAppendAndCollectAfterClose() throws IOException {
```

Comment:
ditto

Reply:
Thanks @chia7712! Agree. Was adding them just to keep it align with the other tests, but can remove for new ones, 👍🏽